gemrb/plugins/IDSImporter/IDSImporter.h

```diff
@@ -24,20 +24,14 @@
 #include "SymbolMgr.h"
 #include "Strings/StringView.h"
 
+#include <utility>
 #include <vector>
 
 namespace GemRB {
 
 class IDSImporter : public SymbolMgr {
 private:
-	struct Pair {
-		int val;
-		std::string str;
-		
-		Pair(int val, std::string str)
-		: val(val), str(std::move(str))
-		{}
-	};
+	using Pair = std::pair<int, std::string>;
```

Comment:
could you use `HeterogeneousStringKey`? It looks like we have a bunch of `stricmp` that can go away.

Reply:
bah, my mistake. that isn't part of the key. Maybe we need a sase insensitive `char_traits` after all.